string part_name
geometry_msgs/PoseStamped part_pose
const_pose[] const_pose_list
